packages/ui/components/checkbox-group/src/LionCheckboxIndeterminate.js

```diff
@@ -259,7 +287,7 @@ export class LionCheckboxIndeterminate extends LionCheckbox {
   /** @param {import('lit-element').PropertyValues } changedProperties */
   updated(changedProperties) {
     super.updated(changedProperties);
-    if (changedProperties.has('indeterminate')) {
+    if (changedProperties.has('indeterminate') || changedProperties.has('forceSyncIndeterminate')) {
```

Comment:
We also could get rid of `forceSyncIndeterminate` completly in the code to improve readability
```suggestion
    if (changedProperties.has('indeterminate') ||  || changedProperties.has('checked')) {
```

Reply:
Thank you, updated as you suggested with some comments above (as, checking `checked` to sync indeterminate might not seem obvious).